pages/author/[slug].js

```diff
@@ -0,0 +1,130 @@
+            {author.totalStory} Stories by {author.name}
+          </div>
+          {author.posts.data.map((post) => {
+            post = post.attributes;
+            return (
+              <div
+                className="flex flex-col basis-[30%] justify-center hover:animate-jump-card -translate-y-6 m-2.5 flex-[1_1_0%] z-0 border shadow-md rounded-xl"
+                key={post.id}
+              >
+                <div className="rounded-t-lg overflow-hidden relative  pt-[52%]">
+                  <div className="absolute inset-0">
+                    <div className="relative">
+                      <>
+                        <a href={"/post/" + post.slug}>
+                          <Image
+                            layout="responsive"
+                            objectFit="contain"
+                            width={376}
+                            height={190}
+                            src={"/../public/apple-touch-icon.png"}
+                            alt={""}
+                          />
+                        </a>
+                      </>
+                    </div>
+                  </div>
+                </div>
+                <div className="flex flex-col justify-between flex-[1_0_0%] px-2.5 py-5 sm:p-5 rounded-b-lg">
+                  <a href={"/post/" + post.slug} className="text-black text-xl">
+                    {post.title}
+                  </a>
+                  <div>
+                    <MarkdownView
+                      className="text-gray-800 mt-5 text-sm line-clamp-3"
+                      markdown={post.content}
+                    />
+                    <div className="pt-4">
+                      <div className="flex flex-row ">
+                        <div className="text-gray-500 flex">
+                          <span>{post.publishedAt}</span>
+                          <span className=" after:content-['\00B7'] after:mx-1 "></span>
+                          <span>{post.readingTime}</span>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            );
+          })}
+        </div>
+      </div>
+    </section>
+  );
+};
+
+export async function getStaticPaths() {
+  const Authors = await fetchAuthor();
+  const paths = Authors.data.map((Author) => ({
+    params: { slug: Author.attributes.slug },
+  }));
+  return {
+    paths: paths,
+    fallback: false,
+  };
+}
+
+export async function getStaticProps(context) {
+  const slug = context.params.slug;
+
+  if (!slug) {
+    throw new Error("Slug not valid");
+  }
+
+  var author = await fetchAuthor(slug);
+  author = author.data.attributes;
+
+  author["totalStory"] = author.posts.data.length;
+  for (let i = 0; i < author.posts.data.length; i++) {
+    const post = author.posts.data[i].attributes;
+    const formattedDate = await formateDate(post.publishedAt);
+    post.publishedAt = formattedDate;
```

Comment:
Can directly apply `await formateDate(post.publishedAt)` here as there is no further use of `formattedDate`